lib/collection/request.js

```diff
@@ -155,14 +155,15 @@ _.assign(Request.prototype, /** @lends Request.prototype */ {
      * @param {Object=} options
      * @param {Boolean} options.ignoreCase When set to "true", will ensure that all the header keys are lower case.
      * @param {Boolean} options.enabled Only get the enabled headers
-     *
+     * @param {Boolean} options.multiValue When set to "true", duplicate header values will be stored in an array
```

Comment:
@codenirvana - does thing break, if we allow multi value by default?

Reply:
Yes, `getHeaders` method is used for `aws4` auth which might fail if duplicates are present.
Not sure about the current state, possibly having issues with current implementation as well 🤔